### PR TITLE
price + token on osmosis

### PR DESCRIPTION
**Pull request by enriquesouza**
This adds the tokens for all cosmos. We will need to add the tokens. I am trying to find a dynamic list so we don't have to do it manually.

https://github.com/cosmos/chain-registry/blob/master/osmosis/assetlist.json 

Osmosis Tests
https://www.mintscan.io/osmosis/address/osmo1a9fkyr59ue6syu9cjnm4e4npgxxxe9llkujhja

Cosmos Tests
https://www.mintscan.io/cosmos/tx/38A67140E5008DB416CDC0E17FAA79485002A91A6AF87854D6AC82C0DA8E25FB

### CODE REVIEW

File: frontend/src/token-store.ts [hunk 1]
```diff
@@ -262,6 +262,17 @@ class Token {
     isNativeToken: true,
   };
 
+  static gaiaChainUSDC: CoinMeta = {
```
Comment (johnnyluo) [inline frontend/src/token-store.ts:265]: Looks like this one is the same as `CoinList`

Comment (enriquesouza) [reply frontend/src/token-store.ts:265]: We will have to refactor them both. We must have only one.

Comment (enriquesouza) [reply frontend/src/token-store.ts]: Removed

**Review by johnnyluo — APPROVED** [review]
LGTM

**Pull request merged**